joblib/test/test_func_inspect_special_encoding.py

```diff
@@ -7,4 +7,3 @@ def big5_f():
     """
     # ����
```

Comment:
Did not knew we had such comments 😅

Reply:
Me neither, live and learn every day :wink: